lib/bindings/HTTPBindings.js

```diff
@@ -72,7 +72,9 @@ function parseData(req, res, next) {
     config.getLogger().debug(context, 'Parsing payload [%s]', payload);
```

Comment:
CNR entry missing.

In addition, I understand that PR should include unit test to cover the new functionality. In addition, documentation should be updated (although in this case I'm not sure if the piece of information to update belongs to the IOTA repository or to the library).